src/communication/azext_communication/manual/custom.py

```diff
@@ -285,85 +285,96 @@ def communication_rooms_remove_participants(client, room_id, participants):
         elif attachments is None or attachment_types is None:
             raise CLIError('Number of attachments and attachment-types should match.')
         elif len(attachments) != len(attachment_types):
             raise CLIError('Number of attachments and attachment-types should match.')
         else:
-            attachments_list = [
-                __get_attachment_content(attachments[i], attachment_types[i])
-                for i in range(len(attachments))
-            ]
-
-        message = EmailMessage(
-            sender=sender,
-            content=email_content,
-            recipients=EmailRecipients(
-                to=to_address,
-                cc=[] if recipients_cc is None else [EmailAddress(email=r) for r in recipients_cc],
-                bcc=[] if recipients_bcc is None else [EmailAddress(email=r) for r in recipients_bcc]),
-            importance=importance,
-            reply_to=reply_to_address,
-            disable_user_engagement_tracking=disable_tracking,
-            attachments=attachments_list,
-        )
-
-        return client.send(message)
-    except HttpResponseError:
-        raise
-    except Exception as ex:
-        sys.exit(str(ex))
-
-
-def communication_email_get_status(client, message_id):
-    try:
-        return client.get_send_status(message_id)
+            all_attachments = attachments[0].split(',')
+            all_attachment_types = attachment_types[0].split(',')
+            for i in range(len(all_attachments)):
+                attachments_list.append(__get_attachment_content(all_attachments[i], all_attachment_types[i]))
+
+        message = {
+            "content": {
+                "subject": subject,
+                "plainText": text,
+                "html": html
+            },
+            "recipients": {
+                "to": None if recipients_to is None else 
+                    [
+                        {"address": recipient} for recipient in recipients_to[0].split(',')
+                    ],
+                "cc": None if recipients_cc is None else 
+                    [
+                        {"address": recipient} for recipient in recipients_cc[0].split(',')
+                    ],
+                "bcc": None if recipients_bcc is None else 
+                    [
+                        {"address": recipient} for recipient in recipients_bcc[0].split(',')
+                    ]
+            },
+            "replyTo": None if reply_to is None else
+                [
+                    {"address": reply_to}
+                ],
+            "attachments": None if attachments_list is None else
+                [
+                    json.loads(attachment) for attachment in attachments_list
+                ],
+            "senderAddress": sender,
+            "userEngagementTrackingDisabled": disable_tracking,
+            "headers": None if importance is None else
+                {
+                    "x-priority": importance
```

Comment:
Here you should take customer input and convert it into a value between 1-5. See previous related comment.

Reply:
Got it